### PR TITLE
White spacing, use createAsEffectType helper

### PR DESCRIPTION
**Pull request by abroz**

### CODE REVIEW

File: src/internal/io.js [hunk 1]
```diff
@@ -145,17 +145,19 @@ export function flush(channel) {
   return effect(FLUSH, channel)
 }
 
+const getAsEffectFn = type => effect => effect && effect[IO] && effect[type]
```
Comment (Andarist) [inline src/internal/io.js]: Its not really a `get` but more of a `create` function here. I think `createAsEffectType` is a sufficient name here.

**Comment by Andarist** [comment]
Commented on the code. Aditionally when you change it, it would be nice if you rebase to master + squash those commits.

**Comment by abroz** [comment]
Done.

**Comment by Andarist** [comment]
Thanks

**Pull request merged**